Powershell-PoC-Environment-Setup.ps1

```diff
@@ -20,12 +20,81 @@
     or lawsuits, including attorneys’ fees, that arise or result from the use or distribution of the Sample Code.
     Please note: None of the conditions outlined in the disclaimer above will supersede the terms and conditions contained
     within the Premier Customer Services Description.
+.NOTES
+    AUTHOR(S): Paulo Marques
+    CONTRIBUTOR(S): Preston K. Parsard
+    KEYWORDS: PoC, Deployment, NEW 0.00.00.0008
+    
+    LICENSE:

```

Comment:
Please remove the MIT license, this is already included in the GitHub main page with the files, in a file called LICENSE